### ISSUE TITLE
train loss problem

### ISSUE THREAD

**Issue by tangfayuan** [Daniil-Osokin/lightweight-human-pose-estimation.pytorch#163]
Suddenly there was a small loss during training. What was the reason? batch_size=16, single GPU, are the training results normal? batch_size=16, single GPU, can the training results get normal results? 

Iter: 109200
stage1_pafs_loss:     106.98959228515625
stage1_heatmaps_loss: 38.43838272094727
stage2_pafs_loss:     93.64942905426025
stage2_heatmaps_loss: 33.54309009552002
stage3_pafs_loss:     90.79842319488526
stage3_heatmaps_loss: 32.240625839233395
total_losses_after [0, 0, 0, 0, 0, 0]
Iter: 109300
stage1_pafs_loss:     33.303994522094726
stage1_heatmaps_loss: 11.86858995437622
stage2_pafs_loss:     29.057841224670412
stage2_heatmaps_loss: 10.322876148223877
stage3_pafs_loss:     28.157716484069823
stage3_heatmaps_loss: 9.926036319732667
total_losses_after [0, 0, 0, 0, 0, 0]
Iter: 109400
stage1_pafs_loss:     103.61133445739746
stage1_heatmaps_loss: 37.29932006835938
stage2_pafs_loss:     90.23819976806641
stage2_heatmaps_loss: 32.34993970870972
stage3_pafs_loss:     87.25235107421875
stage3_heatmaps_loss: 31.057738533020018
total_losses_after [0, 0, 0, 0, 0, 0]

**Comment by JunkyByte** [comment]
See this issue https://github.com/Daniil-Osokin/lightweight-human-pose-estimation.pytorch/issues/100

**Comment by zhouxin666251** [comment]
Thank you very much.

**Comment by zhouxin666251** [comment]
thanks
